### ISSUE TITLE
Fuse tkteach with photo import

### ISSUE THREAD

**Issue by lukeforehand** [lukeforehand/deertracker#28]
I am on a mission to improve the model accuracy with more training examples.

I will be collecting thousands more field examples and run them through `--import-photos` for cropping, so I want to integrate the labeling tool directly with this command.

✔ After import, the labeling tool will open the new crops and predicted labels.  When I correct them they will be flagged in the database as "ground_truth".

✔ I will create a tool that exports the "training_img" dataset which is all "ground_truth" crops, as a tarball for training on google colab.

✔ The export tool will have an option to export the database, models, and all photos regardless of "ground_truth", so we can privately share trained models and unlabeled training data, and it's good to have backups :-)

**Comment by lukeforehand** [comment]
@kbrose I'm curious what would happen if I added new training data and --resume training?  Will it keep improving or does new input data mean I should start over at epoch 0 ?

**Comment by kbrose** [comment]
I think it'd be fine, you wouldn't be able to add new classes though, just more examples of the existing classes.
